src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -82,15 +83,17 @@ export const Finish = (props: OwnProps) => {
         Curious to learn more? Try these next steps!
       </p>
       <FlexBox margin={ComponentSize.Medium} alignItems={AlignItems.Stretch}>
-        <ResourceCard className="homepage-wizard-next-steps">
-          <SafeBlankLink href="https://github.com/InfluxCommunity/sample-flask/blob/main/app.py">
-            <h4>{CodeTerminalIcon}Sample App</h4>
-          </SafeBlankLink>
-          <p>
-            Play around with our template code of sample app to streamline your
-            own data into InfluxData.
-          </p>
-        </ResourceCard>
+        {showSampleApp && (
+          <ResourceCard className="homepage-wizard-next-steps">
+            <SafeBlankLink href="https://github.com/InfluxCommunity/sample-flask/blob/main/app.py">
+              <h4>{CodeTerminalIcon}Sample App</h4>
+            </SafeBlankLink>
```

Comment:
while you're in here, how about adding events for the links being clicked and then connecting this PR with https://github.com/influxdata/ui/issues/4251 by doing a `closes #4251`

Reply:
Totally!

Reply:
✅